app/storage/storageConfig.js

```diff
@@ -56,17 +68,11 @@ class StorageConfig {
     }
   }
 
-  getChannelBucket(location) {
-    location = location ? location.toLowerCase() : this.defaultLocation;
-    const connection = this.s3ConnectionMap.get(location);
-    return connection ? connection.channelBucket : undefined;
-  }
-
   getResourceBucket(location) {
     location = location ? location.toLowerCase() : this.defaultLocation;
     const connection = this.s3ConnectionMap.get(location);
     return connection ? connection.resourceBucket : undefined;
   }
```

Comment:
Can `getResourceBucket` be removed just like `getChannelBucket`?

Reply:
I had removed it. But the old s3 code uses that one still, so I had to add it back.

Reply:
See proposed change below to s3ResourceHandler.js so that we can remove getResourceBucket entirely.  It shouldn't be used once we change to have a separate bucket per org.